src/services/application.js

```diff
@@ -142,7 +142,7 @@ class CompanyApplicationService {
                     .filter((application) => (stateFilter) ?
                         application.state === stateFilter ||
                         stateFilter.includes(application.state)
-                        : true
+                        : true,
```

Comment:
this comma makes no sense, if you take it out, what is the eslint rule you break?